plugins/inputs/nfsclient/nfsclient.go

```diff
@@ -0,0 +1,326 @@
+package nfsclient
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type NFSClient struct {
+	Fullstat bool
+}
+
+var sampleConfig = `
+  # Read more low-level metrics
+  fullstat = false
```

Comment:
We should either just remove this option and always emit all metrics, the [metric filtering](https://github.com/influxdata/telegraf/blob/master/docs/CONFIGURATION.md#metric-filtering) can be used to limit the output, or since I think thing most of the time the per-op metrics are not needed we could have:
```
operations_include = ["READ", "WRITE"]
operations_exclude = []
```

Reply:
Filtering is important, and at collection time.  From a live host just now:
```
[root@nfsclient01 ~]# wc -l /proc/self/mountstats 
60202 /proc/self/mountstats

[root@nfsclient01 ~]# grep nfs /proc/mounts   -c 
1774
```
Perhaps an extreme case...  I know there are a lot of mounts that I don't want to collect in the first place, in addition to filtering on a per-metric basis.

Reply:
And fine-tuning the operations is also a good idea too, forgot to mention that

Reply:
I think this is also resolved isn't it?